packages/rum/src/viewCollection.ts

```diff
@@ -95,6 +95,12 @@ function newView(
   updateView()
 
   function updateView() {
+    if (viewContext.id !== id) {
+      // The view update may come after the view has ended when the throttled scheduleViewUpdate
+      // function is called right before ending the view.  In this case, we should not send the
+      // updated view event.
+      return
+    }
```

Comment:
It could be clearer to introduce something like `cancelScheduledViewUpdate`
wdyt?

Reply:
A was going for it at first, then I thought this was safer, because it prevents *any* view update when another view is occuring. But indeed it shouldn't happen (we should not call startViewCollection twice for the same lifecycle), so I guess it's fine. Let's make things clearer instead!

Reply:
instead of handling it with a flag, wdyt of having:
```ts
const {throttled: scheduleViewUpdate, cancel: cancelScheduledViewUpdate} = throttle(
```
it would more reflect the intent